client/src/ui/molecules/document-survey/surveys.ts

```diff
@@ -100,4 +89,14 @@ export const SURVEYS: Survey[] = [
     start: 0,
     end: Infinity,
   },
+  {
+    key: SurveyKey.HOMEPAGE_FEEDBACK_2024,
+    bucket: SurveyBucket.HOMEPAGE_FEEDBACK_2024,
+    show: (doc: { mdn_url: string }) => /[^/]+\/$/i.test(doc.mdn_url),
+    src: "https://survey.alchemer.com/s3/8075407/MDN-Homepage-Improvements",
+    teaser: "We are refreshing out homepage and would love",
+    question: "your input",
```

Comment:
Typo. Is this the final copy?

```suggestion
    teaser: "We are refreshing our homepage and would love",
    question: "your input",
```